src/store/asana/index.ts

```diff
@@ -535,7 +588,7 @@ export const useAsanaStore = defineStore("asana", {
       );
     },
 
-    LOAD_SECTIONS(): void {
+    LOAD_SECTIONS(project?: string): void {
```

Comment:
Is this parameter used?

Reply:
This was leftover from failed attempts. I'll remove it.